src/app/reporting/Engine.cpp

```diff
@@ -24,6 +24,7 @@
  */
 
 #include <app/AppBuildConfig.h>
+#include <app/ConcreteClusterPath.h>
```

Comment:
Where is that used?  I don't see it being used.

Reply:
not used, removed.